docs/10-stories.md

```diff
@@ -0,0 +1,168 @@
+# Stories
+
+Source uses [storybook](https://storybook.js.org/) for its API documentation as well as for visual regression testing.
+
+The Source storybook can be run locally using `yarn storybook`
+
+## Story Configuration
+
+1. Story files should either be names `stories.tsx` or `MyComponent.stories.tsx`
+1. Story titles should follow the pattern `${Source | Editorial}/${package}/${component}` e.g. `Source/src-radio/RadioGroup`
+1. Each component should have a Demo story which is designed for documentation purposes and a series of stories detailing specific cases for visual regression testing
+1. Stories should have PascalCase names
+1. The `args` property should be passed to all stories so that the source code is displayed on the docs page
+1. Props should all be commented to autofill the args table
+1. Components should all have JSDoc comments to auto fill the docs page (see [example](#example) below)
+1. Controls should be disabled for all stories except the Demo story
+
+    ```ts
+    export const Story = ...
+
+    Story.parameters = {
+        controls: {
+            disabled: true
+        }
+    }
+    ```
+
+    or
+
+    ```ts
+    export const Story = ...
+
+    Story.story = {
+        parameters: {
+            controls: {
+                disabled: true
+            }
+        }
+    }
+    ```
```

Comment:
What is the difference between these two examples? Do they do exactly the same thing, or are there special circumstances in which it is necessary to use one over the other?

Reply:
They do the same thing. I've included it because a number of stories already use either `Story.parameters = {...}` or `Story.story = {...}` so I was hoping to make it clear that you can add it to an existing statement rather than having to add another. On reflection, that isn't particularly clear from this example so maybe let's remove?

Reply:
Oh I see, thanks for explaining. Yes, maybe we should try to standardise on one or the other? It just makes reading, and knowing what approach to take, much easier for future devs

Reply:
Yep, makes sense. Let's go for `Story.story = {...}` as it's the higher level object.